src/testCombiner.ts

```diff
@@ -90,7 +91,8 @@ export function genTestClass(
   const classNameModified = className.replace(/\$/g, '_');
   // Merge all tests' classAnnotation arrays into a single Set
   const classAnnotations = new Set(([] as string[]).concat(...tests.map((test) => test.classAnnotations)));
-  const uniqueTests = tests[0].id ? deduplicateTestNames(tests) : tests;
+  const standardisedTests: ITestData[] = standardiseTests(tests);
```

Comment:
It probabaly should infer the type automatically:

```suggestion
  const standardisedTests = standardiseTests(tests);
```